src/App.tsx

```diff
@@ -1,39 +1,58 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import { CLock } from './Clock';
```

Comment:
Hm, I don't see this file in your commit. Did you add it?

Plus, make attention naming - should be `<Clock />`

Reply:
I again forgot to, I am so sorry, let me fix it. And I didn't really get what naming you are talking about..

Reply:
Oh yeah, I see now, so careless..